src/Achievement.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import './Achievement.css';
+
+const Achievements = () => {
+  return (
+    <div>
+      <div className="container">
+        <div className="sections-wrapper">
+          <div className="section">
+            <h2 style={{ textAlign: 'center' }}>Achievements</h2>
```

Comment:
```suggestion
            <h2 >Achievements</h2>
```
Move this part to CSS file. We don't usually don't use inline styles if we are using external CSS file